### PR TITLE
wizard(ios) opt-in to screenshots

### PR DESCRIPTION
**Pull request by bruno-garcia**
### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Comment by bruno-garcia** [comment]
Related: https://github.com/getsentry/sentry-docs/issues/5324

File: src/wizard/apple/ios.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ transaction.finish() // Mark the transaction as finished and send it to Sentry
...
 
-> Want to play with some new features? Try out our experimental auto instrumentation for [file I/O](/platforms/apple/performance/instrumentation/automatic-instrumentation/#file-io-instrumentation), [Core Data](/platforms/apple/performance/instrumentation/automatic-instrumentation/#core-data-instrumentation), and [User Interaction Instrumentation](/platforms/apple/performance/instrumentation/automatic-instrumentation/#user-interaction-instrumentation). Experimental features are still a work-in-progress and may have bugs. We recognize the irony.
+> Want to play with some new features? Try out our experimental features for [file I/O](/platforms/apple/performance/instrumentation/automatic-instrumentation/#file-io-instrumentation), [Core Data](/platforms/apple/performance/instrumentation/automatic-instrumentation/#core-data-instrumentation), [User Interaction Instrumentation](/platforms/apple/performance/instrumentation/automatic-instrumentation/#user-interaction-instrumentation), [Screenshots](https://docs.sentry.io/platforms/apple/guides/ios/enriching-events/screenshots/). Experimental features are still a work-in-progress and may have bugs. We recognize the irony.
```
Comment (marandaneto) [inline src/wizard/apple/ios.md:97]: It'd be nice if you add on Android too.

Comment (bruno-garcia) [reply src/wizard/apple/ios.md:97]: Could you please open a PR for Android?

Comment (bruno-garcia) [reply src/wizard/apple/ios.md:97]: There's a tracking issue here for iOS and Unity (cc @bitsandfoxes ): https://github.com/getsentry/sentry-docs/issues/5324

**Pull request merged**